### PR TITLE
Wednesday james links

### PR DESCRIPTION
**Pull request by Jamiezz**
wish there was more substance, though I nixed effort on a slideshow for the main page (did a git restore) and will be making another attempt, wanted to make sure I got a pull request as expected

### CODE REVIEW

File: public/stylesheets/login.css [hunk 1]
```diff
@@ -74,6 +74,13 @@ body {
   padding-bottom: 10px;
 }
 
+#rerouteRegister {
+  box-shadow:inset 0px 1px 0px 0px #d49e28;
+	background:linear-gradient(to bottom, #d0451b 5%, #bc3315 100%);
+	background-color:#d0451b;
+	border-radius:3px;
+	border:1px solid #942911;
```
Comment (genagain) [inline public/stylesheets/login.css:81]: Please de-indent lines 79 through 82

File: public/stylesheets/main.css [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+@import url('https://fonts.googleapis.com/css2?family=Texturina:wght@100&display=swap');
+
+body {
+  font-family: 'Texturina', serif;
+}
+
+/* .container1, .container2, .container3 {
+  width: fit-content;
+  margin: auto;
+} */
```
Comment (genagain) [inline public/stylesheets/main.css]: Please remove these commented out classes.

File: public/stylesheets/main.css [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+}
+
+.child1 {
+  background-image: url('/images/complete.png');
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin: 20px;
+  background-color: #EDE8E7;
+  flex-basis: 33%;
+}
```
Comment (genagain) [inline public/stylesheets/main.css:93]: Please add a new line here.

File: public/stylesheets/main.css [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+.child2 {
+  background-image: url('/images/leper 1.png');
+  background-repeat: no-repeat;
+  background-size:contain;
+  margin: 20px;
+  flex-basis: 33%;
+  background-color: #EDE8E7;
+}
```
Comment (genagain) [inline public/stylesheets/main.css:103]: Please add a new line here.

File: public/stylesheets/main.css [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+  height: 350px;
+}
+
+.juvenile {
+  margin: 20px;
+  background-color: #EDE8E7;
+  flex-basis: 100%;
+}
+
+
```
Comment (genagain) [inline public/stylesheets/main.css]: Please remove these extra two new lines here.

File: public/stylesheets/main.css [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+
+#reroute-login {
+  box-shadow:inset 0px 1px 0px 0px #d49e28;
+	background:linear-gradient(to bottom, #d0451b 5%, #bc3315 100%);
+	background-color:#d0451b;
+	border-radius:3px;
+	border:1px solid #942911;
```
Comment (genagain) [inline public/stylesheets/main.css]: Please de-indent lines 107 to 110

File: public/stylesheets/main.css [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+}
+
+.nav-container-main__xp-container {
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  box-sizing: border-box;
+  border: solid black;
+  background-color: whitesmoke;
+  width: 800px;
+  height: 70px;
+  /* margin: 5px 0; */
```
Comment (genagain) [inline public/stylesheets/main.css]: Please remove this property.

File: public/stylesheets/main.css [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+#reroute-login:disabled {
+  background-color: rgb(155, 155, 155);
+}
+
+#reroute-register {
+  box-shadow:inset 0px 1px 0px 0px #d49e28;
+	background:linear-gradient(to bottom, #d0451b 5%, #bc3315 100%);
+	background-color:#d0451b;
+	border-radius:3px;
+	border:1px solid #942911;
```
Comment (genagain) [inline public/stylesheets/main.css]: Please de-indent lines 119 to 122

**Review by genagain — CHANGES_REQUESTED** [review]
Once you make the requested changes I'll approve it.

File: public/stylesheets/main.css (hunk 2 above)
Comment (Jamiezz) [reply public/stylesheets/main.css]: this one has been deleted, currently working on other requests

File: public/stylesheets/main.css (hunk 4 above)
Comment (Jamiezz) [reply public/stylesheets/main.css:103]: new line added for clarity, working on other requests

File: public/stylesheets/main.css (hunk 5 above)
Comment (Jamiezz) [reply public/stylesheets/main.css]: two lines removed for clarity, working on other requests

File: public/stylesheets/main.css (hunk 7 above)
Comment (Jamiezz) [reply public/stylesheets/main.css]: I have removed the commented out margin: 5px 0; from the class  nav-container-main__xp-container (currently line 154)

File: public/stylesheets/main.css (hunk 8 above)
Comment (Jamiezz) [reply public/stylesheets/main.css]: this no longer looks indented on my end, its currently commented out due to testing

File: public/stylesheets/main.css (hunk 6 above)
Comment (Jamiezz) [reply public/stylesheets/main.css]: this no longer looks indented but is currently commented out due to testing

File: public/stylesheets/main.css (hunk 8 above)
Comment (Jamiezz) [reply public/stylesheets/main.css]: this is no longer indented, but commented out for testing purposes

File: public/stylesheets/main.css [hunk 9]
```diff
@@ -0,0 +1,164 @@
...
+  flex-basis: 100%;
+}
+
+#reroute-login {
+  /* box-shadow:inset 0px 1px 0px 0px #d49e28;
+	background:linear-gradient(to bottom, #d0451b 5%, #bc3315 100%);
+	background-color:#d0451b;
+	border-radius:3px;
+	border:1px solid #942911; */
+}
```
Comment (genagain) [inline public/stylesheets/main.css]: Please remove this class because it is currently commented out.

File: public/stylesheets/main.css [hunk 10]
```diff
@@ -0,0 +1,164 @@
...
+#reroute-login:disabled {
+  background-color: rgb(155, 155, 155);
+}
+
+#reroute-register {
+  /* box-shadow:inset 0px 1px 0px 0px #d49e28;
+	background:linear-gradient(to bottom, #d0451b 5%, #bc3315 100%);
+	background-color:#d0451b;
+	border-radius:3px;
+	border:1px solid #942911; */
```
Comment (genagain) [inline public/stylesheets/main.css]: Please remove this class because it is commented out.

**Review by genagain — CHANGES_REQUESTED** [review]
Thank you for making the changes. Please remove the commented out code and I'll approve it.

**Review by genagain — APPROVED** [review]
Looks good to me! Ship it 🚀

**Pull request merged**